cajita/unit_test/tstFastFourierTransform.hpp

```diff
@@ -30,21 +30,10 @@ using namespace Cajita;
 
 namespace Test
 {
-//---------------------------------------------------------------------------//
-void memoryTest()
-{
-    auto mtype = Experimental::HeffteMemoryTraits<TEST_MEMSPACE>::value;
-    HEFFTE::Memory fft_mem;
-    fft_mem.memory_type = mtype;
-    int size = 12;
-    int nbytes = size * sizeof( double );
-    double* ptr = (double*)fft_mem.smalloc( nbytes, mtype );
-    EXPECT_NE( ptr, nullptr );
-    fft_mem.sfree( ptr, mtype );
-}
 
 //---------------------------------------------------------------------------//
-void forwardReverseTest()
+template <class HostBackendType>
+void forwardReverseTest( bool use_default, bool use_params )
```

Comment:
It strikes me as odd to have both `use_default` and `use_params` here - maybe I am missing something.
The way these are used, I might call them `use_host` and `use_params`, or something of that nature.

Reply:
Fair point - what I meant was whether to use default backend and default parameters